pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraNotes.scala

```diff
@@ -57,7 +58,8 @@ object SierraNotes extends SierraDataTransformer with SierraQueryOps {
       .map {
         case (vf, NotesField(createNote, suppressedSubfields)) =>
           val contents =
-            vf.subfieldsWithoutTags(suppressedSubfields.toSeq: _*)
+            vf.subfieldsWithoutTags(
+                universallySuppressedSubfields.toSeq ++ suppressedSubfields.toSeq: _*)
```

Comment:
Do you need to include `suppressedSubfields` at all now? I wonder if it's worth keeping, or if we should remove it?

Reply:
Ummed and ahhed about it - I've removed it to be clear about the intention.